website/docs/docs/collaborate/explore-projects.md

```diff
@@ -162,8 +162,41 @@ Under the the **Models** option, you can filter on model properties (access or m
 
 </Expandable>
 
+<Expandable alt_header="Trust signals for models" lifecycle="preview">
+
+Trust signal icons offer a quick, at-a-glance view of data health when browsing your dbt models in Explorer. These icons indicate whether a model is **Healthy**, **Caution**, **Degraded**, or **Unknown**. For accurate health data, ensure the resource is up-to-date and has had a recent job run.
+
+Each trust signal icon reflects key data health components, such as whether tests were run successfully, whether the resource has a description, whether it has been built in the past 30 days, and more. 
+
+Access trust signals using the search function or when clicking on **Models** under the **Resource** tab. Hover over or click the trust signal to see detailed information.
+
+#### Healthy 
```

Comment:
If might be beneficial to lead in with a brief sentence
"A resource is considered Healthy if it meets the following criteria:"